### PR TITLE
README refactoring

### PR DESCRIPTION
**Pull request by mlavacca**
# Description

This PR proposes a new structure for the README.

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,36 +1,53 @@
...
+    ·
+    <a href="https://github.com/liqotech/liqo/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%5BFeature%5D">Request Feature</a>
+  </p>
+</p>
 
-Liqo is a framework to enable dynamic sharing across Kubernetes Clusters. You can run your pods on a remote cluster
-seamlessly, without any modification (Kubernetes or your application). 
+## About the project
+Liqo is a platform to enable dynamic and decentralized resource sharing across Kubernetes clusters. Liqo allows to run pods on a remote cluster seamlessly and without any modification of Kubernetes and the applications. With Liqo it is possible to extend the control plane of a Kubernetes cluster across the cluster's boundaries, making multi-cluster native and transparent: collapse an entire remote cluster to a virtual local node, by allowing workloads offloading and resource management compliant with the standard Kubernetes approach.
+
```
Comment (palexster) [inline README.md:32]: I would add here the other distinguishing features that we have: Dynamic Discovery, Built-in cluster interconnection, and multi-cluster services

Comment (mlavacca) [reply README.md:32]: Yes, I wanted to describe the Liqo features in a proper section main features

File: README.md [hunk 2]
```diff
@@ -76,21 +113,37 @@ You can observe that:
...
-To get more information about how to install Liqo in your own clusters and configure it, you can check out the [Liqo Documentation](https://doc.liqo.io/user/).
+## Installation
+
+Liqo can be installed by using Helm. The configuration depends on the cluster type (K8s, K3s, managed) and the provider. Possible scenarios can be found [here](https://doc.liqo.io/user/scenarios/).
 
-## Architecture
+To be completed
 
-Liqo relies on several components:
+## Roadmap
```
Comment (frisso) [inline README.md:122]: There's a file ROADMAP.md in the repo.
Let's keep it, providing a link here to the file.
Do not duplicate content.

File: README.md [hunk 3]
```diff
@@ -76,21 +113,37 @@ You can observe that:
...
-* *Advertisement Operator/Broadcaster*: Those components embed the logic to advertise/accept resources from partner
- clusters and spawn new virtual kubelet instances
-* *Liqonet Operators*: Those operators are responsible to establish Pod-to-Pod and Pod-to-Service connection across 
-partner clusters.
+In the next releases, Liqo plans to have implemented:
 
-...and some others. Check out the architecture [documentation](https://doc.liqo.io/architecture/)
+* Support to more than one cluster to join with
+* Support to Amazon Elastic Kubernetes Service
+* Support to Decentralized governance
```
Comment (frisso) [inline README.md:128]: "Support to Decentralized governance" ==> I thought it was already part of the project.
Be more specific.

File: README.md [hunk 4]
```diff
@@ -76,21 +113,36 @@ You can observe that:
...
 
+## Contributing
+
+All contributors are warmly welcome. If you want to become a new contributor, we are so happy!. Just, before doing it, read the repo's guidelines presented on our [documentation website](https://doc.liqo.io/contributing/).
+
+## Community
+
+|![notification](docs/images/readme/bell-outline-badged.svg) Community Meeting|
+|------------------|
+|Liqo holds weekly community meeting on Monday, 17.30 CET, 9.30 PST. To join the community meeting, follow this [link](https://meet.google.com/dyr-ieso-smu). Convert to your timezone [here](https://www.thetimezoneconverter.com/?t=17:30&tz=UTC%20%28Universal%20Time%20Coordinated%29).|
```
Comment (frisso) [inline README.md:138]:
```suggestion
|Liqo holds weekly community meeting on Monday, 5.30pm CET, 9.30am PST. To join the community meeting, follow this [link](https://meet.google.com/dyr-ieso-smu). Convert to your timezone [here](https://www.thetimezoneconverter.com/?t=17:30&tz=UTC%20%28Universal%20Time%20Coordinated%29).|
```

File: README.md [hunk 5]
```diff
@@ -76,21 +113,36 @@ You can observe that:
...
-Liqo relies on several components:
+## Roadmap
 
-* *Liqo Virtual Kubelet*: Based on [Virtual Kubelet](https://github.com/virtual-kubelet/virtual-kubelet) project, the VK
- is responsible to "masquerade" a foreign Kubernetes cluster.
-* *Advertisement Operator/Broadcaster*: Those components embed the logic to advertise/accept resources from partner
- clusters and spawn new virtual kubelet instances
-* *Liqonet Operators*: Those operators are responsible to establish Pod-to-Pod and Pod-to-Service connection across 
-partner clusters.
+In the next releases, Liqo plans to have implemented:
```
Comment (frisso) [inline README.md:124]:
```suggestion
Planned features for the next release (v0.3, expected mid-July, 2021) are the following:
```

File: README.md [hunk 6]
```diff
@@ -76,21 +113,36 @@ You can observe that:
...
 
-...and some others. Check out the architecture [documentation](https://doc.liqo.io/architecture/)
+* Allow joining with more than one cluster
```
Comment (frisso) [inline README.md:126]:
```suggestion
* Support for deployments spanning across more than two clusters
* Support for a fair scheduling mechanism across multiple clusters
* Support for Amazon Elastic Kubernetes Service
* Support for more-granular permission control over remote cluster resources
```

File: README.md [hunk 7]
```diff
@@ -76,21 +113,36 @@ You can observe that:
...
+* Support to Amazon Elastic Kubernetes Service
```
Comment (frisso) [inline README.md:127]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -76,21 +113,36 @@ You can observe that:
...
+* Support for more-granular permission control over remote cluster resources.
```
Comment (frisso) [inline README.md:128]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,41 +1,78 @@
...
 
-Liqo is a framework to enable dynamic sharing across Kubernetes Clusters. You can run your pods on a remote cluster
-seamlessly, without any modification (Kubernetes or your application). 
-
-Liqo is an open source project started at Politecnico of Turin that allows Kubernetes to seamlessly and securely share resources and services, so you can run your tasks on any other cluster available nearby.
-
-Thanks to the support for K3s, also single machines can participate, creating dynamic, opportunistic data centers that include commodity desktop computers and laptops as well.
-
-Liqo leverages the same highly successful “peering” model of the Internet, without any central point of control. New peering relationships can be established dynamically, whenever needed, even automatically. Cluster auto-discovery can further simplify this process.
+* **Decentralized topology**: peer-to-peer paradigm, hence there is no centralized management entity.
```
Comment (frisso) [inline README.md:62]:
```suggestion
* **Decentralized governance**: peer-to-peer paradigm, without any centralized management entity.
```

File: README.md [hunk 10]
```diff
@@ -1,41 +1,78 @@
...
+* **Clusters discovery**: leverage on three different ways to discover other clusters:
```
Comment (frisso) [inline README.md:63]:
```suggestion
* **Clusters discovery**: leverage on three different ways to discover (and peer to) other clusters:
```

**Comment by frisso** [comment]
LGTM, Mattia. For me we can merge.

**Review by frisso — APPROVED** [review]

**Review by palexster — APPROVED** [review]

File: README.md [hunk 11]
```diff
@@ -51,17 +86,24 @@ export KUBECONFIG=$KUBECONFIG_2
...
 
 ```bash
-kubectl get nodes -o wide
+kubectl get nodes
 ```
 
-Let's use those resources. Deploy the [Google microservice Shop](https://github.com/liqotech/microservices-demo/blob/master/release/kubernetes-manifests.yaml) application via: 
+#### __Use__ the resources
+
+Create a new namespace and label it to tell Liqo that the pods there created are suitable for offloading in the remote cluster.
```
Comment (giorio94) [inline README.md:97]: I would propose something like

```suggestion
Create a new namespace and label it to tell Liqo that the pods created there are suitable for offloading in the remote cluster.
```
or

```suggestion
Create a new namespace and label it to tell Liqo that the pods created in that namespace are suitable for offloading in the remote cluster.
```

**Review by giorio94 — APPROVED** [review]

**Pull request merged**